### PR TITLE
Sidebar Macro Icon support added

### PR DESCRIPTION
**Pull request by Sasmira**
Add support for Sidebar Macro module 

Without module 

With module

### CODE REVIEW

File: css/wfrp4e.css [hunk 1]
```diff
@@ -826,7 +826,7 @@
 
   #sidebar.collapsed {
     width: 50px !important;
-    height: 364px !important;
+    height: 400px !important;
```
Comment (moo-man) [inline css/wfrp4e.css:829]: Wouldn't this apply even if the module is not active?

Comment (Sasmira) [reply css/wfrp4e.css:829]: without the height extension, we lose the last icon :D

Comment (moo-man) [reply css/wfrp4e.css:829]: Right, but with the height extension, what does it look like without the module?

Comment (Sasmira) [reply css/wfrp4e.css:829]: Hoo sorry ! on my first message, you have the 2 first screens without the module but already with the height extension actived

Comment (moo-man) [reply css/wfrp4e.css:829]: Ah sorry I misunderstood, thanks.

Comment (Sasmira) [reply css/wfrp4e.css:829]: No problem I don't speak a lot too :D

**Pull request merged**